nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/bootstrap/BootCommand.java

```diff
@@ -37,7 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-package fish.payara.boot.runtime;
+package com.sun.enterprise.v3.bootstrap;
```

Comment:
We can retain payara package here. And as I said earlier, `config-api` is to place to go. It should be safe to add `glassfish-simple-api` dependency there that BootCommand will need.

Reply:
@pdudits  I've pushed to move the `BootCommand` to `config-api`